backend/app/api/endpoints/knowledge.py

```
@@ -767,3 +772,124 @@ def get_qa_history(
+    )
+
+    return KnowledgeBaseSummaryResponse(
+        knowledge_base_id=kb_id,
+        summary=summary,
+    )
+
+
+@router.post("/{kb_id}/summary/refresh", response_model=dict)
+def refresh_kb_summary(
+    kb_id: int,
+    data: KnowledgeBaseSummaryRefreshRequest = KnowledgeBaseSummaryRefreshRequest(),
```

Ruff (0.14.10)

Do not perform function call `KnowledgeBaseSummaryRefreshRequest` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)